packages/orchestrator-ui-components/src/components/WFOButtonComboBox/WFOButtonComboBox.tsx

```diff
@@ -2,15 +2,16 @@ import React, { FC, useState } from 'react';
 import { EuiButton, EuiPopover, EuiSelectable, EuiSpacer } from '@elastic/eui';
 import { getStyles } from './styles';
 
-export type ComboBoxOption = {
-    itemID: string;
+export type WorkflowComboBoxOption = {
+    workflowName: string;
```

Comment:
Adding this makes the ComboBox less generic and EUI seem to perform a spread op props directly on html elements, resulting in an error like this:
![image](https://github.com/workfloworchestrator/orchestrator-ui/assets/20791917/073f7ad3-26ae-44a3-9f89-cfaefc9b4e40)

Reply:
Fixed